code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -114,6 +114,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	..()
 	flags |= NOREACT // so it doesn't react until you light it
 	create_reagents(chem_volume) // making the cigarette a chemical holder with a maximum volume of 15
+	reagents.add_reagent("nicotine", 15)

```

Comment:
@Iamgoofball would you for a later PR be willing to make a e-cig version that holds slightly more chems and gets more consumed each tick?